src/Utilities/Utils.res

```diff
@@ -799,11 +799,15 @@ let delay = timeOut => {
   })
 }
 let getHeaders = (~uri=?, ~token=?, ~headers=Dict.make(), ()) => {
+  let arrayOfNameAndVersion = String.split(Window.userAgent->OrcaLogger.browserDetect, "-")
   let headerObj =
     [
       ("Content-Type", "application/json"),
       ("X-Client-Version", Window.version),
       ("X-Payment-Confirm-Source", "sdk"),
+      ("Browser-Name", arrayOfNameAndVersion->Array.get(0)->Option.getOr("Others")),
```

Comment:
Please verify the names with @vsrivatsa-juspay

Reply:
Updated